android/hardware/src/main/java/quarks/android/hardware/SensorStreams.java

```diff
@@ -16,14 +16,14 @@ Licensed to the Apache Software Foundation (ASF) under one
 specific language governing permissions and limitations
 under the License.
 */
-package quarks.android.hardware;
+package edgent.android.hardware;
```

Comment:
Android related directory does not seems that changed to edgent

- android/topology/src/main/java/quarks/android/oplet/RunOnUIThread.java
- android/hardware/src/main/java/quarks/android/hardware/SensorStreams.java
- android/topology/src/main/java/quarks/android/topology/ActivityStreams.java
- android/hardware/src/main/java/quarks/android/hardware/runtime/SensorSourceSetup.java
- android/hardware/src/main/java/quarks/android/hardware/runtime/SensorChangeEvents.java